koku/masu/processor/tasks.py

```diff
@@ -65,7 +65,7 @@
 from masu.util.oci.common import deduplicate_reports_for_oci
 
 
-LOG = logging.getLogger(__name__)
+LOG = get_task_logger(__name__)
```

Comment:
Glad to see this used. I was surprised we don't use this currently for celery tasks.